spec/decorators/user_decorator_spec.rb

```diff
@@ -83,25 +83,45 @@
   end
 
   describe '#should_acknowledge_recovery_code?' do
-    it 'returns true when the user has no recovery code and is not omniauthed' do
-      user_decorator = UserDecorator.new(User.new)
-      session = { omniauthed: false }
+    context 'user has no recovery code and is not omniauthed' do
+      context 'service provider with loa1' do
+        it 'returns true' do
+          user_decorator = UserDecorator.new(User.new)
```

Comment:
I typically prefer the terse and applaud the brief. But that's just my style. Fine to leave as is.